docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Datasources_Agroal.adoc

```diff
@@ -0,0 +1,238 @@
+[[agroal]]
+= Agroal configuration
+
+The Agroal subsystem allows the definition of datasources. Declaring a new datasource consists of two separate steps: provide a JDBC driver and define a datasource that references the driver you installed.
+
+The Agroal subsystem is provided by the https://agroal.github.io/[Agroal] project. For a detailed description of the available configuration properties, please consult the project documentation.
```

Comment:
is there such documentation? I didn't find it.

Reply:
I'm working on it!